### PR TITLE
deployment concerns

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/kubernetes/pages/deployment-concerns.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  router-cpu: 2
+----
+--
+
+
+// Type:procedure
+[id="high-availability"]
+== Creating a high availability site
+
+By default, Kubernetes restarts any router that become unresponsive.
```
Comment (jcliffor) [inline modules/kubernetes/pages/deployment-concerns.adoc:59]: By default, Kubernetes restarts any router that become**s**

Comment (jcliffor) [reply modules/kubernetes/pages/deployment-concerns.adoc:59]: Should this not refer to downstream ie. OCP?

File: modules/kubernetes/pages/deployment-concerns.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+If you encounter router pod issues consider xref:increased-traffic[] before increasing the number of routers.
```
Comment (jcliffor) [inline modules/kubernetes/pages/deployment-concerns.adoc:60]: If you encounter router pod issues**,**

File: modules/kubernetes/pages/deployment-concerns.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+To address high availability concerns, increase the number of routers to 2 as follows:
+
```
Comment (jcliffor) [inline modules/kubernetes/pages/deployment-concerns.adoc:63]: Both of the following steps render as step 1 which I think is not intentional?

File: modules/kubernetes/pages/deployment-concerns.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  name: "my-site"
+  router-cpu: 2
+----
+--
+
+
+// Type:procedure
+[id="high-availability"]
+== Creating a high availability site
+
```
Comment (jcliffor) [inline modules/kubernetes/pages/deployment-concerns.adoc:58]: The title is not rendering correctly in the Github viewer

File: modules/kubernetes/pages/deployment-concerns.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
++
+--
+[source, bash]
+----
+$ skupper init --routers 2
+----
+--
+
+. If you are using YAML,set the number of routers to `2` by setting the `routers` value:
++
```
Comment (jcliffor) [inline modules/kubernetes/pages/deployment-concerns.adoc:74]: missing space before set

**Review by jcliffor — CHANGES_REQUESTED** [review]

File: modules/kubernetes/pages/deployment-concerns.adoc (hunk 1 above)
Comment (pwright) [reply modules/kubernetes/pages/deployment-concerns.adoc:59]: no, OCP is only one option

**Pull request merged**